modules/developers-guide/pages/rest-using.adoc

```diff
@@ -301,10 +304,109 @@ include::example$result/rest_curl_change_column.result[]
 --
 ====
 
+To add a set to a table, specify the data type `set` along with the data type of the
+set contents, such as `text`:
+
+[tabs]
+====
+cURL command (/v2)::
++
+--
+[source,shell]
+----
+include::example$rest/curl_add_set_to_table.sh[]
+----
+--
+
+Result::
++
+--
+[source,plaintext]
+----
+include::example$result/rest_curl_add_set_to_table.result[]
+----
+--
+====
+
+To add a list to a table, specify the data type `list` along with the data type of the
+set contents, such as `text`.
+The difference between a set and a list is that a set is unordered, whereas a list
+is ordered:
+
+[tabs]
+====
+cURL command (/v2)::
++
+--
+[source,shell]
+----
+include::example$rest/curl_add_list_to_table.sh[]
+----
+--
+
+Result::
++
+--
+[source,plaintext]
+----
+include::example$result/rest_curl_add_list_to_table.result[]
+----
+--
+====
+
+To add a map to a table, specify the data type `map` along with the data type of the
+map key and the may value:
+
+
+[tabs]
+====
+cURL command (/v2)::
++
+--
+[source,shell]
+----
+include::example$rest/curl_add_map_to_table.sh[]
+----
+--
+
+Result::
++
+--
+[source,plaintext]
+----
+include::example$result/rest_curl_add_map_to_table.result[]
+----
+--
+====
+
+To add a tuple to a table, specify the data type `tuple` along with the data type of the
+each item in the tuple.
+A tuple can consist of two or more values that can be retrieved together.
+
+[tabs]
+====
+cURL command (/v2)::
++
+--
+[source,shell]
+----
+include::example$rest/curl_add_tuple_to_table.sh[]
+----
+--
+
+Result::
++
+--
+[source,plaintext]
+----
+include::example$result/rest_curl_add_tuple_to_table.result[]
+----
+--
+====
```

Comment:
Also worth a mention: UDTs.
It's not possible yet to create a UDT via the REST API; but if the UDT already exists, it's possible to create a table that references it, and insert and retrieve data.

Reply:
I was kind of waiting on if a method to create UDTs would be written for REST. I'm concerned about the amount of direct CQL that keeps creeping into REST documents. I already have to show the CREATE INDEX. But that is why you don't see a UDT example yet.

Reply:
I've created a schema query and insert for UDT. Because of a current bug, I don't have a query for retrieving data at the moment.